mscore/tourhandler.cpp

```diff
@@ -348,7 +348,7 @@ void TourHandler::positionMessage(QList<QWidget*> widgets, QMessageBox* mbox)
                   widgetsBox.setRight(qMax(widgetsBox.right(), bottomRight.x()));
                   }
             }
-      if (!set)
+      if (!set) 
```

Comment:
minor quibble: remove this trailing whitespace, because it is unncessary and cause it makes your commit look like you changed a lined when you really didn't change the code.

Reply:
Added new commit to solve